pom.xml

```
@@ -181,6 +181,7 @@
         <mockito.version>4.11.0</mockito.version>
         <netty.version>4.1.100.Final</netty.version>
         <openai.java.version>0.12.0</openai.java.version>
+        <retrofit2.converter-jackson.version>2.9.0</retrofit2.converter-jackson.version>
```

they shares the same version, let's use the main one
```suggestion
        <retrofit2.version>2.9.0</retrofit2.converter-jackson.version>
```

---

Done. Plz help review again @pan3793